chap-cheri-riscv.tex

```diff
@@ -678,103 +678,57 @@ \subsubsection{Non-Compressed Instructions Affected by Capability Encoding
 \subsection{Compressed Instructions}
 \label{subsection:compressed-instructions}
 
-\rwnote{Ensure that this text is up-to-date after our revisions to the
-  encoding-mode mechanism.}
-
 The compressed instruction extension (extension \texttt{C}) is now
-routinely used by the RISC-V gcc compiler to improve code density.  It seems
-likely that the \texttt{C}ompressed extension will become mandatory for
-the \texttt{G}eneral configuration (which is currently
-\texttt{IMAFD}).
+routinely used to improve code density.  While
+the \texttt{C}ompressed extension is not mandatory for
```

Comment:
The C and G markup here is actually not very noticeable in the rendered version and should probably be axed.  We don't follow that convention anywhere else in this chapter either.

Reply:
I've removed this unusual markup and also compressed this down to a single sentence as the two remaining ones were redundant.